src/Breadcrumbs.php

```diff
@@ -64,6 +64,10 @@ public function parse() {
 			return;
 		}
 
+		if ( empty( $this->args ) ) {
+			return;
+		}
```

Comment:
Sao lại bị empty được em nhỉ?

Reply:
khi breadcrumb bị disable thì chỗ này bị empty anh, do trước khi gọi parse thì nó merge với $atts
![Screenshot from 2022-05-05 16-47-44](https://user-images.githubusercontent.com/19669012/166899639-295cf9f2-54d0-4859-adca-6e9d44aeab5c.png)

Reply:
@rilwis khi breadcrumb bị disable thì chỗ này bị empty anh, chắc do trước khi gọi parse thì nó merge với $atts mà biến này rỗng.
![Screenshot from 2022-05-05 16-47-44](https://user-images.githubusercontent.com/19669012/166900496-d069c786-6162-4ced-935d-22fb381a2dd3.png)

Reply:
À, hiểu rồi. Khi disable thì hàm `setup()` ko chạy, nên `$this->args` ko được set.